### ISSUE TITLE
UI: (4.D) Inventory Page: Vulnerabilities Link not visible (CRASM-349)

### ISSUE THREAD

**Issue by hawkishpolicy** [cisagov/XFD#382]
# 🐛 Summary #
After the “All Vulnerabilities” link, focus moves to the upward arrow SVG icon but the focus is not visible to keyboard users.  Focus must be visible when users have navigated to the Sort arrow icon button.
Provide a visible border or outline around links, menu items, and other actionable elements when in focus.

## Any helpful log output or screenshots ##

![image](https://github.com/cisagov/XFD/assets/123108455/8eb09898-fdb7-4caa-9d75-777d81251d47)
